registration-processor/mosip-stage-executor/src/main/java/io/mosip/registration/processor/stages/executor/MosipStageExecutorApplication.java

```diff
@@ -0,0 +1,80 @@
+package io.mosip.registration.processor.stages.executor;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.core.env.MutablePropertySources;
+
+import io.mosip.kernel.core.exception.ExceptionUtils;
+import io.mosip.registration.processor.core.abstractverticle.MosipVerticleAPIManager;
+import io.mosip.registration.processor.stages.executor.config.StagesConfig;
+import io.mosip.registration.processor.stages.executor.util.StageClassesUtil;
+
+/**
+ * The Class MosipStageExecutorApplication.
+ */
+public class MosipStageExecutorApplication {
+	
+	/** The Constant regProcLogger. */
+	private static final Logger regProcLogger = LoggerFactory.getLogger(MosipStageExecutorApplication.class);
+
+	/**
+	 * main method to launch external stage application.
+	 *
+	 * @param args the arguments
+	 */
+	public static void main(String[] args) {
+		regProcLogger.info("Starting mosip-stage-executor>>>>>>>>>>>>>>");
+		//This context is closed after deploying the stages
+		try (AnnotationConfigApplicationContext stageInfoApplicationContext = new AnnotationConfigApplicationContext(
+				new Class<?>[] { StagesConfig.class });) {
+			StagesConfig stagesConfig = stageInfoApplicationContext.getBean(StagesConfig.class);
+			MutablePropertySources propertySources = stagesConfig.getCloudPropertySources();
```

Comment:
Only mosip-stage-executor bootstrap property file will be loaded. Any stage specific property to be specified can be specified in registration-processor.properties file. It is not advised to add a stage specific property to mosip-stage-executor bootstrap properties, because same executor needs to be used for all stage groups.